### DISCUSSION TITLE
TotalCommander not working

### DISCUSSION THREAD

**Discussion by andreas-wittig** [dracula/dracula-theme#737] [Q&A] [answered]
The dark theme does not work with this configuration in TC 10.50. As you see, everything stays white.
![image](https://user-images.githubusercontent.com/33422371/179160480-e8694378-a49f-4be6-b5c0-c4764c83de38.png)

**Comment by luxonauta** [comment, marked as answer]
Thanks for your interest in providing this feedback. ✨

But there is a better place to share the same, which is the project repository, where you can talk to the theme maintainers.
